scripts/zones/Lower_Jeuno/npcs/Chululu.lua

```diff
@@ -1,115 +1,149 @@
-            player:addQuest(JEUNO,tpz.quest.id.jeuno.COLLECT_TARUT_CARDS);
-            player:addItem(card,5);
-            player:messageSpecial(ID.text.ITEM_OBTAINED,card);
+            player:addQuest(JEUNO,tpz.quest.id.jeuno.COLLECT_TARUT_CARDS)
+            player:addItem(card,5)
+            player:messageSpecial(ID.text.ITEM_OBTAINED,card)
         end
     elseif (csid == 200) then
-        player:addTitle(tpz.title.CARD_COLLECTOR);
-        player:addFame(JEUNO, 30);
-        player:tradeComplete();
-        player:completeQuest(JEUNO,tpz.quest.id.jeuno.COLLECT_TARUT_CARDS);
+        player:addTitle(tpz.title.CARD_COLLECTOR)
+        player:addFame(JEUNO, 30)
+        player:tradeComplete()
+        player:completeQuest(JEUNO,tpz.quest.id.jeuno.COLLECT_TARUT_CARDS)
     elseif (csid == 199 and option == 0) then
-        player:addCharVar("RubbishDay_prog", 1);
-        player:setCharVar("RubbishDay_day", VanadielDayOfTheYear()); -- new vanadiel day
+        player:addCharVar("RubbishDay_prog", 1)
+        player:setCharVar("RubbishDay_day", VanadielDayOfTheYear()) -- new vanadiel day
     elseif (csid == 198 and option == 0) then
-        player:addQuest(JEUNO,tpz.quest.id.jeuno.RUBBISH_DAY);
-        player:addKeyItem(tpz.ki.MAGIC_TRASH);
-        player:messageSpecial(ID.text.KEYITEM_OBTAINED,tpz.ki.MAGIC_TRASH);
-        player:setCharVar("RubbishDay_prog",0);
-        player:setCharVar("RubbishDay_day",0);
-    elseif (csid == 197) then
-        if (player:getFreeSlotsCount() == 0) then
-            player:messageSpecial(ID.text.ITEM_CANNOT_BE_OBTAINED,13083);
+        player:addQuest(JEUNO,tpz.quest.id.jeuno.RUBBISH_DAY)
+        player:addKeyItem(tpz.ki.MAGIC_TRASH)
+        player:messageSpecial(ID.text.KEYITEM_OBTAINED,tpz.ki.MAGIC_TRASH)
+        player:setCharVar("RubbishDay_prog",0)
+        player:setCharVar("RubbishDay_day",VanadielDayOfTheYear())
+	elseif (csid == 10110 or csid == 10112 or csid == 10113) and option == 0 then -- ALL_IN_THE_CARDS started, repeated, or additional cards given
+        local rand = math.random(1,4)
+        local card = 0
+
+        if (rand == 1) then
+            card = 559 -- Tarut: Death
+        elseif (rand == 2) then
+            card = 562 -- Tarut: Hermit
+        elseif (rand == 3) then
+            card = 561 -- Tarut: King
         else
-            player:addGil(GIL_RATE*6000);
-            player:messageSpecial(ID.text.GIL_OBTAINED,GIL_RATE*6000);
-            player:addItem(13083);
-            player:messageSpecial(ID.text.ITEM_OBTAINED,13083);
-            player:setCharVar("RubbishDayVar",0);
-            player:addFame(JEUNO, 30);
-            player:completeQuest(JEUNO,tpz.quest.id.jeuno.RUBBISH_DAY);
+            card = 558 -- Tarut: Fool
         end
+
+		if npcUtil.giveItem(player, {{card, 5}}) then
+            player:addQuest(JEUNO,tpz.quest.id.jeuno.ALL_IN_THE_CARDS)
+		player:setCharVar("AllInTheCards_date", getMidnight())
+        player:setLocalVar("Cardstemp", 1)
```

Comment:
indentation went a little odd around here due to tabs.

Reply:
Weird how that got formatted. Silly me, assuming EditorConfig took care of it all. On my end, all lined up.

Reply:
May be some leftover tabs in the file, and it's lining up in your editor because the "indent space" for a tab is set to be equal width to four actual spaces. End result is it looks okay on your screen, but if that tab space amount is different on a different display (ie: here on GitHub) it'll no longer line up.

Reply:
> Silly me, assuming EditorConfig took care of it all. 

it actually should have. dunno why it didn't.